internal/pkg/services/ske/utils/utils.go

```diff
@@ -246,6 +247,45 @@ func ConvertToSeconds(timeStr string) (*string, error) {
 	return utils.Ptr(strconv.FormatUint(result, 10)), nil
 }
 
+// Merge new Kubeconfig into existing Kubeconfig. If it doesn´t exits, creates a new one
+func MergeKubeConfig(pathDestionationKubeConfig, contentNewKubeConfig string) error {
+	if contentNewKubeConfig == "" {
+		return fmt.Errorf("no data to merge. the new kubeconfig is empty")
+	}
+
+	newConfig, err := clientcmd.Load([]byte(contentNewKubeConfig))
+	if err != nil {
+		return fmt.Errorf("error loading new kubeconfig: %w", err)
+	}
+
+	// if the destionation kubeconfig does not exist, create a new one
+	if _, err := os.Stat(pathDestionationKubeConfig); os.IsNotExist(err) {
+		return WriteConfigFile(pathDestionationKubeConfig, contentNewKubeConfig)
+	}
+
+	existingConfig, err := clientcmd.LoadFromFile(pathDestionationKubeConfig)
+	if err != nil {
+		return fmt.Errorf("error loading existing kubeconfig: %w", err)
+	}
+
+	for name, authInfo := range newConfig.AuthInfos {
```

Comment:
Use [`maps.Copy`](https://pkg.go.dev/maps#Copy) function. (Has the same implementation but saves a bit of lines)